runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -9828,12 +9666,21 @@ J9::X86::TreeEvaluator::vectorizedHashCodeHelper(TR::Node *node, TR::DataType dt
    generateLabelInstruction(TR::InstOpCode::label, node, residueEndLoopLabel, deps, cg);
    }
 
+   if (nonZeroOffset)
+      {
+      cg->stopUsingRegister(address);
+      cg->decReferenceCount(node->getChild(1));
+      }
+   else
+      {
+      cg->recursivelyDecReferenceCount(node->getChild(1));
```

Comment:
I believe Child(1) could only be an `iconst` at this point, in which case `recursivelyDecReferenceCount` is overkill.  Just use `decReferenceCount` to save some pathlength.